data/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "command": "./scan ../data/federal-2015-03-15.csv --scan=inspect,tls,analytics --analytics=../data/dap-2015-05-29.csv --debug --output=../reports/federal-2015-06-07",
+  "end_time": "2015-06-09T18:52:41Z",
+  "start_time": "2015-06-09T18:52:33Z"
+}
```

Comment:
I've added a `meta.json` to memorialize the scan parameters, after adding support in https://github.com/18F/domain-scan/pull/18.